tools/src/vendoring/config/react-native-async-storage-scoped-storage-ios.patch

```diff
@@ -0,0 +1,306 @@
+--- ios/RNCAsyncStorage.h
++++ ios/RNCAsyncStorage.h
+@@ -31,12 +31,12 @@
+ 
+ @property (nonatomic, readonly, getter=isValid) BOOL valid;
+ 
++// NOTE(nikki): Added to allow scoped per Expo app
```

Comment:
Are you Nikki? 😂

Reply:
haha, i just want to keep the credit for Nikki 😂
https://github.com/expo/react-native/commit/c7d3f6d12ed566bd5700c8ec0d6b5b82018207a4